tests/lib/ignored-paths.js

```diff
@@ -244,6 +244,12 @@ describe("IgnoredPaths", () => {
             assert.strictEqual(ignoredPaths.ignoreFileDir, path.dirname(ignoreFilePath));
         });
 
+        it("should set the common ancestor directory of cwd and ignorePath", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, ignorePath: ignoreFilePath, cwd: path.resolve(ignoreFilePath, "../../../testcwd") });
+
+            assert.strictEqual(ignoredPaths.getBaseDir(), path.resolve(ignoreFilePath, "../../.."));
+        });
+
```

Comment:
Can you add tests for `ignoredPaths.contains` when `ignoreFilePath` and `cwd` are siblings? It would be useful to make sure that the default ignore patterns (e.g. `node_modules`) and custom ignore patterns both work.

Reply:
Probably I added it.